tools/prow-auto-bumper/file.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+func cdToRootDir() error {
+	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
+	output, err := cmd.Output()
+	if err != nil {
+		return err
+	}
+	d := strings.TrimSpace(string(output))
+	log.Printf("Changing working directory to %s...", d)
+	return os.Chdir(d)
+}
+
+func call(cmd string, args ...string) error {
+	c := exec.Command(cmd, args...)
+	c.Stdout = os.Stdout
+	c.Stderr = os.Stderr
+	return c.Run()
+}
+
+// update all tags in a byte slice
+func updateAllTags(pv *PRVersions, content []byte, imageFilter *regexp.Regexp) ([]byte, string, []string) {
```

Comment:
should this be ?
```suggestion
func updateAllTags(pv *PRVersions, content string, imageFilter *regexp.Regexp) (string, string, []string) {
```

Reply:
[]byte helps better with substring extraction